modules/core/server/services/influx.server.service.js

```diff
@@ -44,12 +46,12 @@ exports.getClient = function(callback) {
  * influxdb (camelCase)
  * @param {Object} fields - key: value pairs will be saved in influxdb as field
  * key: field value
- * @param {number|Date} [fields.time=new Date()] - time of measurement with precision ms
+ * @param {number|Date} [fields.time=new Date()] - time of measurement with precision nanosecond
```

Comment:
I think we can force `Date` and let Influx take care of it. If we ever need `ns`, it's gonna be a place for refactoring, which is fine.